docker/config/webapp.env.template

```diff
@@ -2,6 +2,7 @@ SECRET_KEY=insert_random_key
 DJANGO_DEV=True
 DJANGO_DEBUG=True
 DATABASE_URL=postgres://pontoon:asdf@postgresql/pontoon
+MANUAL_SYNC=True
```

Comment:
if it wants to be `True` by default in a dev env, what about making it equal `DEBUG` or somesuch by default